{"name": "# Verify searched word in Shop by Brand, Numbers, 1 Shot \"1 SHOT\" is here", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a href=\"/collections/1-shot\" class=\"site-nav--link\" data-meganav-type=\"child\" tabindex=\"-1\">...</a> is not clickable at point (414, 303). Other element would receive the click: <main class=\"wrapper main-content\" role=\"main\">...</main>\n  (Session info: chrome=88.0.4324.150)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\007_shopbybrand_numbers_oneshot.py\", line 8, in shopbybrand_numbers_oneshot\n    context.app.main_page.shopbybrand_numbers_oneshot(txt)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\autocareparts_gene_kayakov\\pages\\main_page.py\", line 162, in shopbybrand_numbers_oneshot\n    self.driver.find_element(*ONE_SHOT).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1612492005707, "stop": 1612492007277}, {"name": "Then Verify searched word in Shop by Brand, Numbers, 1 Shot \"1 SHOT\" is here", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <a href=\"/collections/1-shot\" class=\"site-nav--link\" data-meganav-type=\"child\" tabindex=\"-1\">...</a> is not clickable at point (414, 303). Other element would receive the click: <main class=\"wrapper main-content\" role=\"main\">...</main>\n  (Session info: chrome=88.0.4324.150)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\007_shopbybrand_numbers_oneshot.py\", line 8, in shopbybrand_numbers_oneshot\n    context.app.main_page.shopbybrand_numbers_oneshot(txt)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\autocareparts_gene_kayakov\\pages\\main_page.py\", line 162, in shopbybrand_numbers_oneshot\n    self.driver.find_element(*ONE_SHOT).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1612492007277, "stop": 1612492009969}], "start": 1612492004420, "stop": 1612492012230, "uuid": "f34b8ec6-d095-49bc-a0b4-a10f83e58aa2", "historyId": "d7e82d6d0cd5984ffd60b7add56374ac", "fullName": "# Verify searched word in Shop by Brand, Numbers, 1 Shot \"1 SHOT\" is here: # Verify searched word in Shop by Brand, Numbers, 1 Shot \"1 SHOT\" is here", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# Verify searched word in Shop by Brand, Numbers, 1 Shot \"1 SHOT\" is here"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}